apps/frontend-pwa/src/lib/token.ts

```diff
@@ -19,6 +16,11 @@ export async function getParticipantToken({
 
   const cookies = nookies.get(ctx)
 
+  // if the cookie no-account exists, return early
+  if (cookies?.['no-account'] === 'true') {
```

Comment:
skip trying to create a new account through LTI if the cookie has been set after deletion